rest/taskana-rest-spring-example/pom.xml

```diff
@@ -132,6 +139,11 @@
 			<artifactId>taskana-web</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
Leave this dependency for now. But please create a task in taskana board to remove this dependency when taskana-simple-history is released to v.0.0.7

Reply:
created TSK-763